### DISCUSSION TITLE
Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by JrG-One** [community/community#77150] [GitHub Education] [answered]
### Hi everyone,

Actually, My sister has used my Educational Institute Id to take benifits of Student developer pack on her Github Account

now i want to apply for Github Octernship, But I'm unable to Active My Student developer pack on my Github Account 
please help

**Comment by manik-007** [comment]
Hello JrG-One 
Github Student Developer Pack is used by the educational mail, you can just remove the mail from her github account and add it to yours. Then you can use the guthub developer pack in your account.

suggestion : just like your sister has used your id to redeem developer pack you can use someone else who does not have any interest in open source, in this way you can take benefit of developer pack without compromising your sister pack.

I hope this helps you :)

**Comment by JrG-One** [comment, marked as answer]
Hello @manik-007 ,

Could you please guide me through the step-by-step process to remove it from my sister's account since she no longer requires it.

And thank you for suggesting the method as well.

**Comment by manik-007** [comment]
@JrG-One I would love to guide you.
Go to your profile < profile settings < under public email (email settings) < remove the mail id 
then add the mail id on your github account following the same procces, this time you have to add that mail instead of removing. 
In this way you will add the mail which is having github developers pack and can enjoy the pack on your account.

If you find this helpful in any respect, then kindly accept it as answer, then it will become my first ever answer on github (+_+).
If Faced any issue then feel free to ask.
